pages/linux/ip-route-get.md

```diff
@@ -0,0 +1,24 @@
+# ip route get
+
+> Get a single route - this command gets a single route to a destination and prints its contents exactly as the kernel sees it.
```

Comment:
```suggestion
> Get a single route to a destination and print its contents exactly as the kernel sees it.
```

Reply:
A brief explanation for the author:

The original description seems to be posted verbatim from the manual. Here at tldr though, the description should be adapted to fit [the style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood).